proxy/http_headers.go

```diff
@@ -72,6 +73,20 @@ func addHeaders(r *http.Request, cfg config.Proxy) error {
 	return nil
 }
 
+// addRequestID Set header with name from `cfg.RequestID` to UUID value
+//
+func addRequestID(r *http.Request, p *HTTPProxy) {
+	if p.Config.RequestID != "" {
```

Comment:
I see. Then just inline the couple lines into the HTTP proxy and use the integration test to test it.